knitware-bots/barr-e/spine/1_idf_clean.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+clear
+rm -rf build
+
+export WORKLOAD_PRESET=spine-workloads-barr-e
+export ROBOTICK_PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/.."
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)